src/pages/index.tsx

```diff
@@ -1,6 +1,8 @@
 import Head from "next/head";
 import { Inter } from "next/font/google";
-import styles from "@src/styles/Home.module.css";
+import styles from "@/styles/Home.module.css";
```

Comment:
Since the Home module css has been deleted, I don't think we need to import this anymore mas.

Reply:
Ah noted thanks Dan

Reply:
oh I see, I just realized that you deleted the root styles directory.